doc/docker/Dockerfile-distribution

```diff
@@ -3,6 +3,8 @@
 ARG DISTRIBUTION_IMAGE=docker_app
 FROM ${DISTRIBUTION_IMAGE} as builder
 
+ENV GOOGLE_TRANSLATE_API_KEY=MustBeSetInOrderToDumpAssetsOfEzPlatformAutomatedTranslationBundle
```

Comment:
I don't get it... the GOOGLE_TRANSLATE_API_KEY env variable must be set to some random value ( doesn't need to be a valid key ) in order for the build to work?

Reply:
Yes. At this stage (building the distribution image with all distribution files) the variable need to be just set to whatever value.
Later when the application starts, the valid key must be provided.

Reply:
so, it seems like the env variable must exist in order for the automated-translation bundle to be taken into account when creating assetics